PSGSuite/Public/Drive/Remove-GSDriveFile.ps1

```diff
@@ -0,0 +1,66 @@
+function Remove-GSDriveFile {
+    <#
+    .SYNOPSIS
+    Deletes a Drive file
+
+    .DESCRIPTION
+    Deletes a Drive file
+
+    .PARAMETER FileId
+    The unique Id of the file to get
+
+    .PARAMETER User
+    The email or unique Id of the owner of the Drive file
+
+    Defaults to the AdminEmail user
+
+    .EXAMPLE
+    Remove-GSDriveFile -User user@domain.com -FileId '1rhsAYTOB_vrpvfwImPmWy0TcVa2sgmQa_9u976'
+
+    Deletes the file with ID 1rhsAYTOB_vrpvfwImPmWy0TcVa2sgmQa_9u976 from the user user@domain.com's Drive
+    #>
+    [cmdletbinding(SupportsShouldProcess = $true, ConfirmImpact = "High")]
+    Param
+    (
+        [parameter(Mandatory = $true, Position = 0)]
+        [String[]]
+        $FileId,
+        [parameter(Mandatory = $false, ValueFromPipelineByPropertyName = $true)]
+        [Alias('Owner', 'PrimaryEmail', 'UserKey', 'Mail')]
+        [string]
+        $User = $Script:PSGSuite.AdminEmail
+    )
+    Begin {
+        if ($User -ceq 'me') {
+            $User = $Script:PSGSuite.AdminEmail
+        }
+        elseif ($User -notlike "*@*.*") {
+            $User = "$($User)@$($Script:PSGSuite.Domain)"
+        }
+        $serviceParams = @{
+            Scope       = 'https://www.googleapis.com/auth/drive'
+            ServiceType = 'Google.Apis.Drive.v3.DriveService'
+            User        = $User
+        }
+        $service = New-GoogleService @serviceParams
+    }
+    Process {
+        try {
+            if ($PSCmdlet.ShouldProcess("Deleting File Id(s) '$FileId' from user '$User'")) {
```

Comment:
Let's get this inside the `foreach` loop and add in the Write-Verbose statements before and after, i.e.:

```powershell
foreach ($file in $FileId) {
    if ($PSCmdlet.ShouldProcess("Deleting File Id '$file' from user '$User'")) {
        Write-Verbose "Deleting File Id '$file' from user '$User'"
        $request = $service.Files.Delete($file)
        $request.Execute()
        Write-Verbose "File Id '$file' successfully deleted from user '$User'"
}
```

Also - if the request type doesn't support Team Drives, let's pull that commented line out, otherwise let's uncomment it so that Team Drives interaction can be supported to (it doesn't impact anything, as far as I'm aware, to leave it enabled)